### PR TITLE
Refactor init

### PR DESCRIPTION
**Pull request by kwy95**
While testing kw I ran `kw init` on my kernel folder and lost my configurations, with this change we avoid losing any special configurations.

I also included some small changes to the init files.

Signed-off-by: Rubens Gomes Neto <rubens.gomes.neto@usp.br>

### CODE REVIEW

File: src/init.sh [hunk 1]
```diff
@@ -18,12 +18,21 @@ function init_kw()
     exit 0
   fi
 
+  if [[ -f "$PWD/$name" ]]; then
+    if [[ "$*" =~ --?f(orce)? || $(ask_yN "$name already exists, do you wish to overwrite it?") =~ '1' ]]; then
+      mv "$PWD/$name" "$PWD/$name.old"
+    else
+      say 'Initialization aborted!'
+      exit 0
```
Comment (rodrigosiqueira) [inline src/init.sh:26]: Add a test that validates this condition.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
In general, lgtm; however, I want to see @magalilemes feedback since she is working on init.

**Review by magalilemes — APPROVED** [review]
Looks good to me. Although if this is merged before #404, I'll have to make a few adjustments, but no big deal.

File: src/init.sh (hunk 1 above)
Comment (kwy95) [reply src/init.sh:26]: Done!

**Comment by rodrigosiqueira** [comment]
In this case, merged to the unstable branch. Thanks

**Pull request closed without merging**